### PR TITLE
feat(cell-guide): add tissue descriptions to TissueCards

### PR DESCRIPTION
**Pull request by atarashansky**
## Reason for Change

- #5102 
- Descriptions are parsed from the `latest` UBERON OBO file.

## Testing
 - added e2e test
 - tests pass
 - tested locally

### CODE REVIEW

**Comment by brianraymor** [comment]
> Descriptions are parsed from the latest UBERON OBO file.

Why can't the pinned version be used?

File: frontend/src/views/CellGuide/components/TissueCard/index.tsx [hunk 1]
```diff
@@ -36,6 +48,14 @@ export default function TissueCard(): JSX.Element {
     return () => window.removeEventListener("resize", onResize);
   }, []);
 
+  const [descriptionUberon, setDescriptionUberon] = useState<string>("");
```
Comment (tihuan) [inline frontend/src/views/CellGuide/components/TissueCard/index.tsx:51]: Feels like we might not need this state and can just use rawDescriptionUberon directly?

**Review by tihuan — APPROVED** [review]
FE code looks great! Thanks for adding this quickly, Alec!

Would you be able to add a 150 char version of tissue description as part of the SEO JSON file too? 

Please see Brian Raymor's question before merging :pray: 

Thank you!

**Comment by atarashansky** [comment]
> FE code looks great! Thanks for adding this quickly, Alec!
> 
> Would you be able to add a 150 char version of tissue description as part of the SEO JSON file too?
> 
> Please see Brian Raymor's question before merging 🙏
> 
> Thank you!

You'd think I would have learned by now to use pinned versions! Will update this now. Thanks for reminding me again

File: frontend/src/views/CellGuide/components/TissueCard/index.tsx (hunk 1 above)
Comment (tihuan) [reply frontend/src/views/CellGuide/components/TissueCard/index.tsx:51]: @atarashansky PTAL thanks!

Comment (tihuan) [reply frontend/src/views/CellGuide/components/TissueCard/index.tsx:51]: Actually never mind, for SEO I'm going to fetch the description at build time, so I'll just remove this completely! Thanks!

**Pull request merged**